src/dotnet-core-uninstall/Shared/Commands/UninstallCommandExec.cs

```diff
@@ -108,7 +110,7 @@ private static void TryIt(IEnumerable<Bundle> bundles)
             Console.WriteLine();
             Console.WriteLine(string.Format(
                 LocalizableStrings.DryRunHowToDoItMessageFormat,
-                string.Join(" ", Environment.GetCommandLineArgs())));
+                $"{Process.GetCurrentProcess().MainModule.FileName} {string.Join(" ", Environment.GetCommandLineArgs().Skip(1))}"));
```

Comment:
I'd suggest this as the full path isn't usually necessary when invoking command line tools:

```suggestion
                $"{Path.GetFileName(Process.GetCurrentProcess().MainModule.FileName)} {string.Join(" ", Environment.GetCommandLineArgs().Skip(1))}"));
```

Reply:
However, I think this would be confusing if the user is _not_ running the executable in its directory.

Reply:
I'm not sure if we are going to ship a single executable or an MS installer to install it to some `PATH` location.

Reply:
I feel the output to be too long and not generally helpful.  I would not expect users to "copy and paste" the output to re-run with the `--do-it` option; they'd do what most developers would do: "hit up and type `--do-it`".

However, I don't have a strong opinion.

Reply:
Likewise, I would also be fine with a simple (and static) message of `To actually perform the uninstall, use the '--do-it' option.`  The command's sitting in my shell history, and I know where to put the option :)

Reply:
Done.